hamocc/mo_aufr_bgc.F90

```diff
@@ -454,6 +455,16 @@ subroutine aufr_bgc(kpie,kpje,kpke,ntr,ntrbgc,itrbgc,trc,kplyear,kplmon,kplday,o
     if (use_shelfsea_res_time .and. lread_shelfage) then
       call read_netcdf_var(ncid,'shelfage',locetra(1,1,1,ishelfage),2*kpke,0,iotype)
     endif
+    if (use_river2omip) then
+      call read_netcdf_var(ncid,'tdoc_lc',locetra(1,1,1,itdoc_lc),2*kpke,0,iotype)
+      call read_netcdf_var(ncid,'tdoc_hc',locetra(1,1,1,itdoc_hc),2*kpke,0,iotype)
+      if (use_cisonew) then
```

Comment:
There is the possibility to switch on C-isotopes but to restart the model from a restart file that does not contain the C13/C14 fields. This will not work as it is done here - please move this block into the if-block `if (use_cisonew .and. lread_iso) then` (starting in line 468). The variable `lread_iso` is only true if the restart file actually contains the C14/14 fields.